R/utilities-units.R

```diff
@@ -347,14 +348,18 @@ getUnitsEnum <- function() {
         getUnitsForDimension(dimension = dimension)
       },
       error = function(cond) {
-        warning(messages$errorLoadingUnitsForDimension(dimension, cond))
+        errors <- c(errors, dimension)
```

Comment:
You need to do super-assignment here:

```suggestion
        errors <<- c(errors, dimension)
```